pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2174,7 +2174,7 @@ cdef class BusinessHour(BusinessMixin):
 
         # adjust by business days first
         if bd != 0:
-            if self._prefix.startswith("C"):
+            if self._prefix.startswith("c"):
                 # GH#30593 this is a Custom offset
                 skip_bd = CustomBusinessDay(
```

Comment:
as above, if we know if it's custom business day, won't it be `'C'`?

Reply:
I'm not sure, is it always custom business day? Some tests for `CustomBusinessHour` , e.g.
`test_custom_businesshour_weekmask_and_holidays`
`test_us_federal_holiday_with_datetime`
go to line 2179, , which is why I changed capitalisation.

Reply:
right sorry, this is `CustomBusinessHour`, so `'c'` should always be sufficient

all good then, resolving this comment